.goreleaser.yml

```diff
@@ -108,7 +108,7 @@ dockers:
       - hack/addons/mindthegap-helm-registry/repos.yaml
     build_flag_templates:
       - "--platform=linux/amd64"
-      - "--pull"
```

Comment:
Was this done to fix an issue or just speed up the build?
(Want to make sure we have some history of why this changed)

Reply:
good catch - this was removed to do something else.